pyproject.toml

```diff
@@ -7,7 +7,7 @@ description = "Python bindings for the Plex API."
 readme = "README.rst"
 requires-python = ">=3.9"
 keywords = ["plex", "api"]
-license = {file = "LICENSE.txt"}
+license = {text = "BSD-3-Clause"}
```

Comment:
Should this just be

```suggestion
license = "BSD-3-Clause"
```

Reply:
That's not yet supported by setuptools unfortunately. Using the legacy `license.text` is the next best thing for the time being.

_There is a open PR in setuptools to add full support for it. That one depends on some other (unrelated) work though, so it's still some time until it's added._